utilities/create_site.sh

```diff
@@ -30,4 +30,17 @@ git add index.html
 echo "<html><head><script>window.location.replace('/google-cloud-java/${SITE_VERSION_BASE}/apidocs' + location.search)</script></head><body></body></html>" > apidocs/index.html
 git add apidocs/index.html
 
+cd ..
+mvn site:stage --quiet -Djava.util.logging.config.file=logging.properties -DtopSiteURL=http://googlecloudplatform.github.io/google-cloud-java/site/latest/
```

Comment:
I honestly can't tell if we need this or not. The documentation on `-DtopSiteURL` is a little fuzzy, so I re-stage here just in case.